impl/client/src/main/javascript/web/dojo/pentaho/common/TemplatePicker.js

```diff
@@ -33,8 +33,16 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
             templateSelectedCallback: null,
 
             updatePageArrows: function () {
-              this.prevSetBtn.set('disabled', this.pageNo == 0);
-              this.nextSetBtn.set('disabled', (this.pageNo + 1) * 6 >= this.templates.length);
+              var isFirstPage = this.pageNo == 0;
+              var isLastPage = (this.pageNo + 1) * 6 >= this.templates.length;
+              this.prevSetBtn.set('disabled', isFirstPage);
+              this.nextSetBtn.set('disabled', isLastPage);
+              if(this.prevSetBtn.domNode.getAttribute("tabindex") === "-1") {
+                this.nextSetBtn.domNode.focus();
+              }
```

Comment:
Is this detecting whether the button is disabled? If so, why not either use the `isFirstPage` variable directly, or else `this.prevSetBtn.get('disabled')`?

Reply:
Yes, added this to prevent focus going out of dialog . Will address the change.